src/teleop/jetson/jetson_teleop.py

```diff
@@ -308,15 +352,18 @@ def main():
 
     ros.Subscriber("joystick", Joy, drive.teleop_drive_callback)
     ros.Subscriber("xbox/ra_control", Joy, arm.ra_control_callback)
-    ros.Subscriber("ra/mode", String, arm.ra_mode_callback)
     ros.Subscriber("ra_slow_mode", Bool, arm.slow_mode_callback)
     ros.Subscriber("xbox/sa_control", Joy, arm.sa_control_callback)
     ros.Subscriber("brushless_ra_data", MotorsStatus, arm.brushless_encoder_callback)
     ros.Subscriber("brushed_ra_data", JointState, arm.brushed_encoder_callback)
 
+    # Arm Mode Services
+    ros.Service("change_ra_mode", ChangeArmMode, arm.ra_mode_service)
+    ros.Service("change_sa_mode", ChangeArmMode, arm.sa_mode_service)
+
     # Publish joint states for Moveit at 10Hz
     while not ros.is_shutdown():
-        arm.publish_joint_states()
+        # arm.publish_joint_states()
```

Comment:
So why is this commented out?

Reply:
It's not needed, publishes come from callbacks to separate SA from RA. We should just remove this though and replace this while loop with a rospy.spin() though I think

Reply:
I commented this because as it stands publishing to joint_states flood the terminal on the jetson with warnings. We will need to publish this if we ever get moveit working.

Reply:
Ok yeah I figured. If you wanna, you can make it based on a boolean and topic so we can opt to having it publish vs not during the mission